django_cloud_deploy/cli/prompt.py

```diff
@@ -1136,6 +1136,33 @@ def _validate(self, s: str):
                               '"test@example.com"').format(s))
 
 
+class TakeSurveyPrompt(object):
```

Comment:
OK, maybe it would make sense at this point to put this in it's own file, maybe with the logic to open the web browser?

Reply:
It is using the private method "_binary_prompt" in prompt.py. So maybe also make that method public?

Reply:
Ahhh....yes, you could make that public. I think that it make sense to put this in a new file because I can imagine wanting to do the survey for `update` to but maybe only the first time or something.

Reply:
Done.

Reply:
I think I will add survey prompt for update on a follow up PR. Right now I have not figured out how to prompt only once.